dayPlanner.js

```diff
@@ -14,84 +14,39 @@ for (var i = 1; i <= 15; i++) {
             if (element.style.display === "none" || element.style.display === "") {
                 element.style.display = "block";
             } else {element.style.display = "none";}
+            if (activeFuncNum !== null && activeFuncNum !== undefined) {
+                let deleteButton = "SİL"
+                let addButon = "EKLE";
+                let td = "td" + activeFuncNum;

```

Comment:
- işlenen `td` targetını çağırıp ona dair işlemleri yapmak için `td` değişkeni tanımlıyoruz.